### DISCUSSION TITLE
Extract monai features from ohif in radiology active learning workflow

### DISCUSSION THREAD

**Discussion by orenculmination** [Project-MONAI/MONAI#7052] [Q&A]
We set up the radiology active learning workflow with OHIF (https://github.com/Project-MONAI/tutorials/blob/main/monailabel/monailabel_radiology_spleen_segmentation_OHIF.ipynb). What is the best way to extract the features monailabel generates when invoked from the OHIF GUI? I would like to be able to extract them into a separate database. Does OHIF save the features? is it easier to get it from OHIF or directly from the monailabel server, forking its output somehow? Thank you!

**Comment by KumoLiu** [comment]
Hi @tangy5, could you please share some comments on this one?
Thanks in advance!
